rules/windows/file/file_event/file_event_win_susp_double_extension.yml

```
@@ -16,7 +16,7 @@ references:
     - https://cloud.google.com/blog/topics/threat-intelligence/cybercriminals-weaponize-fake-ai-websites
```

```suggestion
    - https://cloud.google.com/blog/topics/threat-intelligence/cybercriminals-weaponize-fake-ai-websites
    - https://vipre.com/blog/svg-phishing-attacks-the-new-trick-in-the-cybercriminals-playbook/
```
Just add the `.svg` reference